android/quest/src/main/java/org/smartregister/fhircore/quest/ui/profile/ProfileViewModel.kt

```diff
@@ -80,6 +81,8 @@ constructor(
   private val listResourceDataStateMap =
     mutableStateMapOf<String, SnapshotStateList<ResourceData>>()
 
+  val shouldReloadData = MutableLiveData<Boolean>()
```

Comment:
Why do we need a new LiveData just to refresh the profile when dialog is dismissed?